### PR TITLE
Streamlining the book introductions 

### PR DESCRIPTION
**Pull request by taroth21**
### Description

Streamlining the book introductions. Removed the info about the book structure (as this can be seen from the TOCs), added missing intros to two books and made sure each book has an abstract. 

### To which product versions do the changes apply?

The first column is to be filled by the requester, the second column is to be filled by the doc team after the changes have been backported to the maintenance branches.

| Code 15     | Backport Done
| ----------  | ----------
| [x] 15 SP3  | *(no backport necessary)*
| [ ] 15 SP2  | [ ]
| [ ] 15 SP1  | [ ]
| [ ] 15 SP0  | [ ]

| Code 12     | Backport Done
| ----------  | ----------
| [ ] 12 SP5  | [ ]
| [ ] 12 SP4  | [ ]
| [ ] 12 SP3  | [ ]
| [ ] 12 SP2  | [ ]

### CODE REVIEW

File: xml/book_sles_containers.xml [hunk 1]
```diff
@@ -22,6 +22,7 @@
   <xi:include href="common_copyright_gfdl.xml"/>
   <abstract>
    <para>
+    This guide provides an introduction to the &suse; container ecosystem.
```
Comment (lproven) [inline xml/book_sles_containers.xml:25]: I do not like this word -- it sounds too much like marketing to me. However, any replacement I can think of would be a whole sentence.

**Review by lproven — COMMENTED** [review]
I think I am missing something here -- I see a lot of deleted text, but nothing to replace it. Is this a WIP or does it pair up with one of the other PRs?

**Comment by taroth21** [comment]
> I think I am missing something here -- I see a lot of deleted text, but nothing to replace it. Is this a WIP or does it pair up with one of the other PRs?

No, I really removed a lot of text about the book structure, as the book structure (which parts etc.) is visible from the TOC.

File: xml/book_sles_containers.xml (hunk 1 above)
Comment (taroth21) [reply xml/book_sles_containers.xml:25]: I just moved over to the abstract what was already written elsewhere (before the sentence was in container_intro.xml). I think we have another chance to fix it when the work on the guide continues.

**Pull request merged**